ras_party/error_handlers.py

```diff
@@ -29,7 +29,7 @@ def http_error(error):
 
 @blueprint.app_errorhandler(RasError)
 def ras_error(error):
-    logger.info(error.to_dict())
+    logger.exception(error.to_dict(), status=error.status_code, **error.kwargs)
```

Comment:
Do you need to log error.to_dict() - this will be in the exc_info logged by the excpeiton level log handler.